src/components/SearchBar/SearchBar.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { fireEvent, render, screen, waitFor } from '@testing-library/react';
```

Comment:
I suggest to use userEvent instead in future
https://testing-library.com/docs/user-event/intro
`fireEvent dispatches DOM events, whereas user-event simulates full interactions, which may fire multiple events and do additional checks along the way.`
But as I remember tests are not in scope of this task at all, so that's really good!